### ISSUE TITLE
Assigning IP workflow is unworkable. No matter the direction it is done.

### ISSUE THREAD

**Issue by phobiadhs** [digitalocean/netbox#1759]
### Issue type
[x ] Feature request 
[x ] Bug report      
[ ] Documentation   

### Environment
* Python version:  3.4.5-4
* NetBox version:  2.2.7

### Description
Currently, adding an IP to an interface requires two active tabs (One in the target prefix to identify an available IP and one on the device to assign the IP.) It requires cutting and pasting and simply isn't efficient or workable. It has made Netbox nearly unusable for my shop.

The old method was set up so that you could create the IP from/within the prefix and assign that IP to a device and interface either by searching or using the drill down method (there was a form for this until recently).

In 2.2.5, the ability to add the next available IP within the prefix was added to the Add IP option within the prefix but that really serves no purpose because you still have to have a second window open to assign the IP you just created.

In short, my request is two fold.

1. When creating an IP within a given prefix using either the add IP method or by selecting from an available block (x available) within the IP list, the ability to assign the created or selected IP to a device be added back. Both the device search and drill down methods were useful and should be included.

2. When adding an IP to a device from the device screen, the ability to select an prefix, and an available IP (or next available within the prefix) within that prefix be added.

Honestly, this is such a big deal for my company that we are willing to resort to flat out bribery to get this functionality back. Scotch? Rye? Beer? Steaks? Say the word.

**Comment by jeremystretch** [comment]
Please see the discussion in #1512. If the current workflow is not sufficient for your needs, NetBox may not be the right solution for you.

**Comment by phobiadhs** [comment]
Have you tried adding an IP to a device? Seriously. Just go through the process and tell me that is an efficient use of clicks and time. It takes twice as long as it did before and is so prone to errors that the entire point of the software flies out the window. Beyond that, there is no capability for bulk IP add either now with this method. This is a regression. Sure, it works with some fooling around but is a regression.
